bucket/wabt.json

```
@@ -0,0 +1,42 @@
+{
+    "homepage": "https://github.com/WebAssembly/wabt",
+    "description": "WABT (we pronounce it \"wabbit\") is a suite of tools for WebAssembly.",
```

Use single quote. No need to escape them.

```suggestion
    "description": "WABT (we pronounce it 'wabbit') is a suite of tools for WebAssembly.",
```

---

Done.